content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Quality Gates
+kind: documentation
+description: Learn how to use Quality Gates in your pipeline.
+is_beta: true
+further_reading:
+- link: "/continuous_integration/tests"
+  tag: "Documentation"
+  text: "Learn about Test Visibility"
+---
+
+## Overview
+
+{{< callout url="#" header="false" >}}
+Quality Gates Rules are in beta.
+{{< /callout >}}
+
+Quality Gates allows you to gate your workflows based on signals in Datadog.
+For example, you can block a Pull Request from being merged if it introduces new flaky tests, even if the tests are passing.
```

Comment:
We can maybe add a link to the Flaky Test Management guide: https://docs.datadoghq.com/continuous_integration/guides/flaky_test_management/!

Reply:
Yeah great point, added